### ISSUE TITLE
ProgressIndicator - In scan mode Backward navigation (Up Arrow keys) information about progress bar is announced but focus stays on the 'Example description'.

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10792]
[Usable]: In scan mode backward navigation(Up arrow keys) information about progress bar is announced but focus stays on the previous element only. 

**User Impact:**
Narrator  user will be confused if in backward navigation (Up arrow keys) focus stays on other control and information about other control is narrated.

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 18999.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/progressindicator
Edge Browser Version:
Microsoft Edge: 44.18999.1.0
Microsoft EdgeHTML: 18.18999

**Usages:**
1.Default Progress Indicator
2.Indeterminate Progress Indicator

**Repro Steps:**
1. Open https://developer.microsoft.com/en-us/fabric#/controls/web/progressindicator in Edge. 
2. Navigate to Progress Indicator under default progress Indicator variant in backward navigation (Up arrow keys) in scan mode(Caps + Space) using up arrow key.
3. Observe the issue.

**Actual Result:**
In scan mode(Caps + Space) backward navigation (Up Arrow keys) information about progress are is announced but focus stays on the previous element (Example description) only.

**Expected Result :**
In scan mode(Caps + Space) backward navigation(Up arrow keys) focus and the information which screen reader is providing should be in sync. If narrator announces the status bar information, focus should be lands on the progress bar.

**Guidelines:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/doc2.aspx?sourcedoc=%7bf98f4235-b6bd-47d3-8da4-f28b9b6fbfa8%7d&action=view&wdAccPdf=0&wdparaid=F2F4701&cid=b3a503cf-a205-4513-b488-8ffa7a60f9ed

**Attachment:**
[Narrator focus and announcment is not in sync.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3715867/Narrator.focus.and.announcment.is.not.in.sync.zip)

Bug 3 in #8388

**Comment by aneeshack4** [comment]
In scan mode, I'm able to repro that the blue focus rect doesn't get placed during backwards nav but does in forward nav:
![image](https://user-images.githubusercontent.com/4161791/71841661-cbb1fb00-3074-11ea-8d1d-6004cce329cf.png)
But I see that this is reproing in this reference example: http://pauljadam.com/demos/progressbar.html so I think this is a general issue rather than one specific to Fabric's progressbar. It might have to do with how Narrator interact with ARIA `role='progressbar'` in scan mode. As far as I can tell, the `[Fabric progressbar](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.base.tsx#L89)` is following the ARIA guidelines: https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#progressbar so I'm not sure what else we can do to solve this especially given that it repros on other progressbars in the web.
